src/c4/test/CMakeLists.txt

```diff
@@ -80,17 +80,20 @@ endif()
 # Tests that expect exactly 4 PE (omit for DRACO_C4=SCALAR builds):
 set( four_pe_tests ${PROJECT_SOURCE_DIR}/tstQuoWrapper.cc )
 list( REMOVE_ITEM test_sources ${four_pe_tests} )
-if( NOT ${DRACO_C4} STREQUAL "SCALAR" )
-  add_parallel_tests(
-    SOURCES   "${four_pe_tests}"
-    DEPS      Lib_c4
-    PE_LIST   "4" )
-  if( NOT DEFINED MPI_MAX_NUMPROCS_PHYSICAL )
-    message(FATAL_ERROR "need max procs in c4/tests/CMakeLists.txt")
+
+if ( NOT APPLE )
```

Comment:
Did you add the "not apple" because of something you discovered testing locally on your mac? Just curious! :)

Reply:
Yes, this was an issue with finding mpi. @KineticTheory will you double check this cmake change for me?

Reply:
I don't think this change should be necessary.  Let's take a closer look at the behavior on your machine.  If there is an issue, it is likely related to `setupMPI.cmake` and should be fixed at that location.

Reply:
@clevelam and I just reviewed the build errors associated with this code.  The core issue is that we haven't written code in `xthi_cpuset.hh` to pull thread affinities for APPLE.